### ISSUE TITLE
Character Drupal 8/9 migration

### ISSUE THREAD

**Issue by steinmb** [jilltxt/machinevision#81]
Migrate node content from Drupal 7 to 8. Clean up data structure and omit not unused data and data structures.

## Data structures
Migrate all fields. All mapping is:`field_name: field_name` except if listed below.

Format: `D8: D7`
* `field_customizable: field_customizable_`
* `field_individual_or_group: field_group`
* `field_optional_notes: field_your_notes`

### Files and media assets
Drupal 7 and 8/9 field names are identical if not mention bellow.

```
Character
    Multimedia
        Field type: Image
        Widget: Media browser
        Field: field_media
        Media types: Image

    Image (Screenshot)
        Field type: Image
        Widget: Media browser
        Field: field_image
        Media types: Image

```

## Needs work

* [x] Remote videos (Youtube, Vimeo etc)
* [x] Individual or group - Data missing
* [x] Customizable - data missing
* [x] Optional notes - Data missing
* [x] Sexuality - keep or drop?
* [x] file and image fields to media entities
* [x] Comments
* [x] Revisions

## How to run test migration

`drush migrate:import uib_node_character` and rollback nodes: `drush migrate:rollback uib_node_character`

## How test

* Verify that fields contain correct values after migration.
* Verify that form and help text is correct looks OK. Do not have to look great but all data needs to be there.

**Comment by steinmb** [comment]
@jilltxt Question: With characters we agreed we should migrate all fields but the taxonomy term ref. `field_sexuality` use the vocabulary: Sexuality that we in #61 should not migrate.

What is correct?

**Comment by jilltxt** [comment]
Ah, that was a mistake, we should KEEP sexuality. I checked and it is used in characters. Sorry about confusion!

**Comment by steinmb** [comment]
Closed issue #86 that also addressed the last pending task here.
